### DISCUSSION TITLE
How can I tell if anything is selected?

### DISCUSSION THREAD

**Discussion by askfiy** [mg979/vim-visual-multi#202] [Q&A] [answered]
Is there an option like `VM.status` to let us see if it has something selected?

Because cmp's &lt;c-n&gt; and &lt;c-p&gt; conflict with the plugin's keys.

So I want to cancel the autocompletion of cmp when the plugin's status = true .

**Comment by mg979** [comment, marked as answer]
You can check `if exists('b:visual_multi')` to see if you are in multiple cursors mode.

In lua you can just check `if vim.b.visual_multi then ...`

**Comment by askfiy** [comment]
Thank you very much
